BSP/Inc/BSP_Precharge.h

```diff
@@ -0,0 +1,20 @@
+#ifndef _BSP_PRECHARGE_H_
+#define _BSP_PRECHARGE_H_
+
+#include "common.h"
+#include "config.h"
+#include <bsp.h>
+
+typedef enum {Array = 0, Motor} board_t;
```

Comment:
Go ahead and name these something like ARRAY_PRECHARGE and MOTOR_PRECHARGE since those names are also used for the contactor enum. We'll change those to be more specific also.